dags/atd_service_bot_index_issues_to_dts_portal.py

```diff
@@ -0,0 +1,66 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from onepasswordconnectsdk.client import Client, new_client
+import onepasswordconnectsdk
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+ONEPASSWORD_CONNECT_HOST = os.getenv("OP_CONNECT")
+ONEPASSWORD_CONNECT_TOKEN = os.getenv("OP_API_TOKEN")
+VAULT_ID = os.getenv("OP_VAULT_ID")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Create/update 'Index' issues in the DTS portal from Github.",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 12, 1),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+}
+
+docker_image = "atddocker/atd-service-bot:production"
+
+REQUIRED_SECRETS = {
+    "KNACK_APP_ID": {
+        "opitem": "Service Bot",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.knackAppId",
+        "opvault": VAULT_ID,
+    },
+    "KNACK_API_KEY": {
+        "opitem": "Service Bot",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.knackApiKey",
+        "opvault": VAULT_ID,
+    },
+    "GITHUB_ACCESS_TOKEN": {
+        "opitem": "Service Bot",
+        "opfield": "shared.githubAccessToken",
+        "opvault": VAULT_ID,
+    },
+}
```

Comment:
I'm a huge fan of getting development vs. production secrets here with this f-string method. 👍